src/buildtools/fslex/Lexing.fs

```diff
@@ -219,7 +219,7 @@ namespace Microsoft.FSharp.Text.Lexing
            LexBuffer<byte>.FromReadFunctions(Some(stream.Read),Some(fun (buf,offset,len) -> stream.AsyncRead(buf,offset=offset,count=len))) 
 
     module GenericImplFragments = 
-        let startInterpret(lexBuffer:LexBuffer<_>)= 
+        let startInterpret(lexBuffer:LexBuffer<_>) = 
```

Comment:
@ncave 

fslex and fsyacc are not managed in this repo, changes to files in these directories need to be directed to the fslexyacc repo: http://github.com/fsprojects/FsLexYacc 

This is noted in the readme: https://github.com/Microsoft/visualfsharp/blob/master/src/buildtools/README-fslexyacc.md